cmd/skaffold/app/cmd/render.go

```diff
@@ -25,13 +25,17 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 )
 
 var (
-	showBuild        bool
-	renderOutputPath string
+	fromBuildOutputFile flags.BuildOutputFileFlag
```

Comment:
We should limit this PR to this flag. 
Please split `offline`, `renderOutputPath`, `showBuild` should go to individual PRs - it is going to be much faster to review.

Reply:
Sorry, I have confused you by ordering those lines alphabetically. The only _new_ flags are `fromBuildOutputFile`/`offline` which I believe should be reviewed together to make sense for the GitOps model. I'm reverting the order so the diff looks sane, and will follow @tstromberg's advice on naming.